pkg/rtc/wrappedreceiver.go

```diff
@@ -29,11 +30,20 @@ func NewWrappedReceiver(receivers []*simulcastReceiver, trackID livekit.TrackID,
 		sfuReceivers = append(sfuReceivers, r.TrackReceiver)
 	}
 
+	codecs := upstreamCodecs
+	// if upstream is opus/red, then add opus to match clients that don't support red
+	if len(codecs) == 1 && strings.EqualFold(codecs[0].MimeType, sfu.MimeTypeAudioRed) {
+		codecs = append(codecs, webrtc.RTPCodecParameters{
+			RTPCodecCapability: webrtc.RTPCodecCapability{MimeType: webrtc.MimeTypeOpus, ClockRate: 48000, Channels: 2, SDPFmtpLine: "minptime=10;useinbandfec=1"},
```

Comment:
Can we make this a constant somewhere, the codec capability for Opus? Looks like it is repeated in a few places.

Reply:
good point